src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -1,12 +1,12 @@
 <jboss-deployment-structure>
-    <deployment>
-        <dependencies>
-            <module name="org.infinispan" services="export" />
-            <module name="org.apache.httpcomponents"/>
-            <module name="org.apache.commons.io"/>
-            <module name="com.fasterxml.jackson.core.jackson-annotations"/>
-            <module name="com.fasterxml.jackson.core.jackson-databind"/>
-            <module name="com.fasterxml.jackson.datatype.jackson-datatype-jsr310"/>
-        </dependencies>
-    </deployment>
+	<deployment>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Tab characters found (this is the first occurrence) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=xml%3AIllegalTabCheck)